deployments/report-server-target-aws.nix

```diff
@@ -1,14 +0,0 @@
-{ accessKeyId, ... }:
-
-with (import ./../lib.nix);
-{
-  network.description = "Cardano SL";
-
-  report-server = { config, pkgs, resources, ... }: {
-    imports = [
-      ./../modules/amazon-base.nix
-    ];
-
-    deployment.ec2.accessKeyId = accessKeyId;
-  };
-}
```

Comment:
№5

We probably want to restore the physical machine spec (AWS, GCE, ...) aspect separation.

Reply:
Filed as https://issues.serokell.io/issue/DEVOPS-456